src/components/ui/OurProjects.jsx

```diff
@@ -0,0 +1,26 @@
+import OurProjectItem from './OurProjectItem';
+import { fetchGraphQL } from '@/lib/graphqlUtils';
+import { ourProjectItem } from '@/queries/ourProjectItem';
+import { ourProjects } from '@/queries/ourProjects';
+
+const OurProjects = async () => {
+  const { ourProjectsHeader } = await fetchGraphQL(ourProjects);
+  const { ourProjectItems } = await fetchGraphQL(ourProjectItem);
```

Comment:
why didn't you do it in one hygraph component

Reply:
easy manage